Modules/ITS/include/ITS/ITSFeeTask.h

```diff
@@ -33,18 +33,23 @@ namespace o2::quality_control_modules::its
 /// \brief ITS FEE task aiming at 100% online data integrity checking
 class ITSFeeTask final : public TaskInterface
 {
-
   struct GBTDdw { //GBT diagnostic word
```

Comment:
Coding Guidelines recommendation: avoid abbreviations. https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html#General_Naming_Rules

Reply:
done. hope the amended variable name meets the requirement.